diff --git a/vendor/mvdan.cc/sh/v3/interp/handler.go b/vendor/mvdan.cc/sh/v3/interp/handler.go
index 484cbce..76b5d9d 100644
--- a/vendor/mvdan.cc/sh/v3/interp/handler.go
+++ b/vendor/mvdan.cc/sh/v3/interp/handler.go
@@ -277,7 +277,7 @@ func pathExts(env expand.Environ) []string {
 	}
 	pathext := env.Get("PATHEXT").String()
 	if pathext == "" {
-		return []string{".com", ".exe", ".bat", ".cmd"}
+		return []string{".com", ".exe", ".bat", ".cmd", ""}
 	}
 	var exts []string
 	for _, e := range strings.Split(strings.ToLower(pathext), `;`) {
diff --git a/vendor/mvdan.cc/sh/v3/syntax/parser.go b/vendor/mvdan.cc/sh/v3/syntax/parser.go
index a4f286b..bdef0f2 100644
--- a/vendor/mvdan.cc/sh/v3/syntax/parser.go
+++ b/vendor/mvdan.cc/sh/v3/syntax/parser.go
@@ -1433,12 +1433,15 @@ func ValidName(val string) bool {
 	if val == "" {
 		return false
 	}
+
+	isValuesRef := strings.HasPrefix(val, "values.")
 	for i, r := range val {
 		switch {
 		case 'a' <= r && r <= 'z':
 		case 'A' <= r && r <= 'Z':
 		case r == '_':
 		case i > 0 && '0' <= r && r <= '9':
+		case r == '.' && isValuesRef:
 		default:
 			return false
 		}
